src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -42,21 +42,13 @@ public function process(ContainerBuilder $container)
         $viewDir = \dirname((new \ReflectionClass('Symfony\Bridge\Twig\Extension\FormExtension'))->getFileName(), 2).'/Resources/views';
         $templateIterator = $container->getDefinition('twig.template_iterator');
         $templatePaths = $templateIterator->getArgument(1);
-        $cacheWarmer = null;
-        if ($container->hasDefinition('twig.cache_warmer')) {
```

Comment:
This service was removed in 5.0 (ref: https://github.com/symfony/symfony/pull/31800/files#diff-af1c832537900d7a594b1a9249348b13L32)